packages/boxel-ui/addon/raw-icons/copy.svg

```diff
@@ -1,6 +1,10 @@
 <svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 14 14">
     <g data-name="Group 13494" transform="translate(1 1)">
-        <rect data-name="Rectangle 5051" width="7.8" height="7.8" rx="1" transform="translate(4.2 4.2)" style="fill:none;stroke:#03c4bf;stroke-linecap:round;stroke-linejoin:round;stroke-width:2px"/>
-        <path data-name="Path 85688" d="M3.8 9.8h-.6A1.2 1.2 0 0 1 2 8.6V3.2A1.2 1.2 0 0 1 3.2 2h5.4a1.2 1.2 0 0 1 1.2 1.2v.6" transform="translate(-2 -2)" style="fill:none;stroke:#03c4bf;stroke-linecap:round;stroke-linejoin:round;stroke-width:2px"/>
+        <rect data-name="Rectangle 5051" width="7.8" height="7.8" rx="1" transform="translate(4.2 4.2)"
+            style="fill:none;stroke:var(--icon-color, #000);stroke-linecap:round;stroke-linejoin:round;stroke-width:var(--icon-stroke-width, 1.5px)" />
```

Comment:
@burieberry I changed this stroke width to be compatible with the trash icon that u changed